metadata/v1.24.0/proposal_1_upgrade_framework.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Multi-step proposal to upgrade mainnet framework, version v1.24.0",
+  "description": "This includes changes in https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.24.0-rc",
```

Comment:
should put the link to the prod release

Reply:
Done, updated it to point to: https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.24.2